micro-rdk/src/common/app_client.rs

```diff
@@ -169,18 +177,20 @@ pub(crate) struct AppSignaling(
 
 impl<'a> AppClient<'a> {
     pub(crate) async fn connect_signaling(&mut self) -> Result<AppSignaling, AppClientError> {
+        let (sender, receiver) = async_channel::bounded::<Bytes>(1);
         let r = self
             .grpc_client
             .build_request(
                 "/proto.rpc.webrtc.v1.SignalingService/Answer",
                 Some(&self.jwt),
                 &self.config.rpc_host,
+                BodyExt::boxed(StreamBody::new(receiver.map(|b| Ok(Frame::data(b))))),
```

Comment:
I'm missing something here. Maybe two things:
- I can't find a `map` on the type that I think `receiver` is (edit: Gautham found it for me).
- I'm not clear on what the addition of this channel is doing or why